tests/test_marathon.py

```diff
@@ -152,6 +152,44 @@ def test_update_pod_raises_dcos_exception_if_deployment_id_missing():
         headers={'marathon-deployment_ID': 'misspelled-field', 'zzz': 'aaa'})
 
 
+def test_kill_pod_instances_executes_successfully():
+    _assert_kill_pod_instances_executes_successfully(
+        pod_id='foo',
+        instance_ids=['instance1', 'instance2'],
+        path='v2/pods/foo::instance',
+        response_json={'some': ['instance', 'status']})
+
+    _assert_kill_pod_instances_executes_successfully(
+        pod_id='/bar baz/',
+        instance_ids=['instance1', 'instance2'],
+        path='v2/pods/bar%20baz::instance',
+        response_json={'some': ['instance', 'status']})
+
+    _assert_kill_pod_instances_executes_successfully(
+        pod_id='foo',
+        instance_ids=['i1', 'i2', 'i3'],
+        path='v2/pods/foo::instance',
+        response_json={'some': ['instance', 'status']})
+
+    _assert_kill_pod_instances_executes_successfully(
+        pod_id='foo',
+        instance_ids=['instance1', 'instance2'],
```

Comment:
the differences here seem to be validating the we can handle variations in the url rather than than the kill command is doing what we expect. Lets have tests that cover url variations (if we don't already) and ones that just test the `pod kill command`